rundeckapp/src/test/groovy/rundeck/services/LogFileStorageServiceSpec.groovy

```diff
@@ -1016,11 +1031,10 @@ class LogFileStorageServiceSpec extends RundeckHibernateSpec implements ServiceU
 
             def exec = new Execution(
                     dateStarted: new Date(),
-                    dateCompleted: null,
+                    dateCompleted: new Date(),
                     user: 'user2',
                     project: 'test',
-                    serverNodeUUID: 'D0CA0A6D-3F85-4F53-A714-313EB57A4D1F',
-                    outputfilepath: '/tmp/file'
+                    serverNodeUUID: 'D0CA0A6D-3F85-4F53-A714-313EB57A4D1F'
```

Comment:
Why did you remove `exec.outputFilePath` in this and the following tests _(L1100, L1164)_?

Reply:
As we changed the way of validating (execution.scheduleExecution == null) if the execution was going to get the file by getting it from "outputfilepath" or by forming the filepath (as showed below) , that part of the test needed to be remove in order to follow its normal steps. Since within those test we do not use a valid execution.scheduleExecution, the tests itself were getting another path which they were not created for.

![image](https://user-images.githubusercontent.com/29233418/194067219-c0a06161-7261-4a2e-bd96-92aede83e484.png)